Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -75,7 +75,6 @@
 - files:
   - borgwalk1.ogg
   - borgwalk2.ogg
-  - borgwalk3.ogg
   license: "CC-BY-SA-4.0"
-  copyright: "Taken from plingativator freesound.org and mixed by https://github.com/MilenVolf"
-  source: "https://freesound.org/people/plingativator/sounds/188917/"
+  copyright: "Taken from IENBA freesound.org and modified by https://github.com/MilenVolf"
+  source: "https://freesound.org/people/IENBA/sounds/697379/"
```

Comment:
Yes, you got it right, both samples from a single audio. First one was just heavily processed.